validator/internal/validator_service_mock.go

```diff
@@ -6,12 +6,11 @@ package internal
 
 import (
 	context "context"
-	reflect "reflect"
-
 	gomock "github.com/golang/mock/gomock"
 	v1 "github.com/prysmaticlabs/prysm/proto/beacon/rpc/v1"
 	grpc "google.golang.org/grpc"
 	metadata "google.golang.org/grpc/metadata"
+	reflect "reflect"
```

Comment:
why does it always do this? can we revert `reflect` to the original position?

Reply:
moved